scripts/tendermint/start.sh

```diff
@@ -3,25 +3,28 @@ set -o errexit -o nounset -o pipefail
 command -v shellcheck > /dev/null && shellcheck "$0"
 
 TM_VERSION=0.21.0
-TM_DIR=$(mktemp -d "${TMPDIR:-/tmp}/tendermint.XXXXXXXXX")
-echo "TM_DIR = $TM_DIR"
+PORT=12345
```

Comment:
Why not allow override?
I was thinking we could run multiple copies of this script to test eg.
0.21 on 12345 and 0.25 on 11223

Reply:
As far as I can see the feature was not needed for now so I simplified the code.

The problem with multiple Tendermint containers is they are all called "tendermint/tendermint:<version>" and stopping the right one gets comlicated. Do we really need that feature? or can we just stop and start a different version?

Reply:
I want to start everything before running jasmine, as I think controlling this from js code is gonna get messy.

If we want to test both 0.21 and 0.25 adaptors work on the same commit, how do you propose we do so?

Reply:
1. set TENDERMINT_ENABLED
2. start tendermint in default version (0.21)
3. run tests
4. stop tendermint
5. start tendermint in new version (0.25)
6. run tests (no JS change or rebuild required since Tendermint URL does not change)
7. stop tendermint

5. requires a code change but I can expose a config variable for the version instead

Reply:
Point 5 works with `TENDERMINT_VERSION=0.25.0 ./scripts/tendermint/start.sh` now